packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/utils.dart

```diff
@@ -123,19 +121,19 @@ class AnimatedFlexLayoutProperties
         begin.mainAxisDimension,
         end.mainAxisDimension,
         animation.value,
-      );
+      )!;
 
   @override
-  MainAxisSize get mainAxisSize => end.mainAxisSize;
+  MainAxisSize? get mainAxisSize => end.mainAxisSize;
 
   @override
-  TextBaseline get textBaseline => end.textBaseline;
+  TextBaseline? get textBaseline => end.textBaseline;
 
   @override
   TextDirection get textDirection => end.textDirection;
 
   @override
-  double get totalFlex =>
+  double? get totalFlex =>
```

Comment:
does this have to be null? lerpDouble is a bit strangely nullable but not an api we control. We should probably create our own version that is null safe as the current behavior seems only reasonable given dart:ui wanted to avoid breaking changes when migrating to null safety.